<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/icons/images/ss.jpg);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="printAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuEdit_E">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="separator"/>
    <addaction name="copyAction"/>
    <addaction name="cutAction"/>
    <addaction name="pasteAction"/>
    <addaction name="selectAllAction"/>
    <addaction name="separator"/>
    <addaction name="datetimeAction"/>
    <addaction name="setFontAction"/>
    <addaction name="setColorAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="updateAction"/>
    <addaction name="aboutSoftwareAction"/>
    <addaction name="helpAction"/>
    <addaction name="aboutWebAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit_E"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAction"/>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
   <addaction name="separator"/>
   <addaction name="copyAction"/>
   <addaction name="cutAction"/>
   <addaction name="pasteAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/images/new.png</normaloff>:/new/icons/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New(&amp;N)</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/images/open.png</normaloff>:/new/icons/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/images/save.png</normaloff>:/new/icons/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/images/save.png</normaloff>:/new/icons/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save As (&amp;A)</string>
   </property>
  </action>
  <action name="printAction">
   <property name="text">
    <string>Print(&amp;P)</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit(&amp;X)</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="text">
    <string>Undo(&amp;U)</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="text">
    <string>Redo(&amp;R)</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/images/copy.png</normaloff>:/new/icons/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy(&amp;C)</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/images/cut.png</normaloff>:/new/icons/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut(&amp;X)</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icons/images/paste.png</normaloff>:/new/icons/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste(&amp;P)</string>
   </property>
  </action>
  <action name="selectAllAction">
   <property name="text">
    <string>Select All(&amp;A)</string>
   </property>
  </action>
  <action name="datetimeAction">
   <property name="text">
    <string>Date(&amp;D)</string>
   </property>
  </action>
  <action name="setFontAction">
   <property name="text">
    <string>Font(&amp;F)</string>
   </property>
  </action>
  <action name="setColorAction">
   <property name="text">
    <string>Color(&amp;L)</string>
   </property>
  </action>
  <action name="updateAction">
   <property name="text">
    <string>Update(&amp;U)</string>
   </property>
  </action>
  <action name="aboutSoftwareAction">
   <property name="text">
    <string>About Software(&amp;S)</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>Help(&amp;E)</string>
   </property>
  </action>
  <action name="aboutWebAction">
   <property name="text">
    <string>About Web(&amp;W)</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>About Qt(&amp;Q)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
